sql/housepartsstorage.sql

```diff
@@ -0,0 +1,18 @@
+CREATE TABLE `housepartsstorage` (
+	`LandSetId` INT UNSIGNED NULL,
+	`HouseId` INT UNSIGNED NULL,
+	`storageId` BIGINT, 
+	`type` INT NULL,
+	`index` INT NULL,
```

Comment:
type and index will need to be queue quoted with \` or these names can be changed to PartType PartIndex etc